app/assets/javascripts/admin/scripts/script_list_view.js

```diff
@@ -16,6 +16,8 @@ import type { RouterHistory } from "react-router-dom";
 const { Column } = Table;
 const { Search } = Input;
 
+const typeHint: APIScriptType[] = [];
```

Comment:
Don't we use the `Array<T>` syntax in most places? Maybe we should keep it consistent. Could we have a linting rule for that?

Reply:
Yeah, we usually have used `Array<T>`, but in my opinion it's quite verbose in comparison to `T[]`. For complex types, `Array<T>` might be easier to read, which is why I'd decide this on a per-case basis. Consistency doesn't add much value in my opinion, since both versions are easy to grasp.

Reply:
Alright. I don't have a strong opinion on this.